src/components/profile/Address.js

```diff
@@ -8,7 +8,7 @@ export function Address() {
   const displayAddress = useMemo(() => {
     if (bnsName.loaded) return bnsName.data;
     if (stxAddress.loaded)
-      return `${stxAddress.data.substr(0, 5)}...${stxAddress.data.substr(
+      return `${stxAddress.data.substring(0, 5)}...${stxAddress.data.substring(
```

Comment:
created an `enhancement/use-stacksutils` using a library that we could think about bringing in to handle some common utility functions such as shortening addresses moving forward